rust/src/lib.rs

```diff
@@ -1780,6 +1811,38 @@ enum NativeScriptEnum {
     TimelockExpiry(TimelockExpiry),
 }
 
+#[derive(Debug, Clone, Eq, Ord, PartialEq, PartialOrd)]
+pub enum ScriptRefEnum {
+    NativeScript(NativeScript),
+    PlutusScriptV1(PlutusScript),
+    PlutusScriptV2(PlutusScript),
+}
```

Comment:
Let's remove `PlutusScriptV2` and leave just one `PlutusScript`. Now that we have discussed that the script version will be within the `PlutusScript` type itself, no reason to have separate enums for them